src/main/java/ch/njol/skript/effects/EffEnchant.java

```diff
@@ -37,71 +37,95 @@
 import ch.njol.skript.util.EnchantmentType;
 import ch.njol.util.Kleenean;
 
-/**
- * @author Peter Güttinger
- */
 @Name("Enchant/Disenchant")
 @Description("Enchant or disenchant an existing item.")
 @Examples({"enchant the player's tool with sharpness 5",
-		"disenchant the player's tool"})
-@Since("2.0")
+		"disenchant the player's tool",
+		"disenchant the player's tool of unbreaking and sharpness",
+		"",
+		"# For enchanted books",
+		"enchant player's tool with stored unbreaking 3",
+		"disenchant stored enchantmentes of player's tool"})
+@Since("2.0, INSERT VERSION (stored, specific disenchant)")
 public class EffEnchant extends Effect {
```

Comment:
I'm not really a big fan of the syntax here. I don't think it's necessary to support stored enchantments in this expression. Using ExprEnchantments should be good enough :)

Reply:
Pattern has been updated, check it out and let me know :)